### PR TITLE
Move styles to seperate file, fixes #1

### LINKED ISSUE #1: Cannot scroll vertically
overflow should be overflow-x

### PR DESCRIPTION
**Pull request by Toby56**

### CODE REVIEW

**Comment by Toby56** [comment]
Good?

File: styles.css [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-      form { background: #000; padding: 3px; position: fixed; bottom: 0; width: 100%; }
-      form input { border: 0; padding: 10px; width: 90%; margin-right: .5%; }
-      form button { width: 9%; background: rgb(130, 224, 255); border: none; padding: 10px; }
-      #messages { list-style-type: none; margin: 0; padding: 0; width: 100%;}
-      #messages li { padding: 5px 10px; width: 100%;}
-      #messages li:nth-child(odd) { background: #eee; }
-      #messages li:nth-child(even) { background: #fafafa; }
-      #users { position: fixed; top: 15px; right: 15px; background-color: rgba(0, 0, 0, 0.4); padding: 10px; border-radius: 5px;}
+body, ul, div, form, input { margin: 0; padding: 0; box-sizing: border-box; }
+body { font: 13px 'Roboto', Helvetica, Arial; overflow: auto;}
```
Comment (platy11) [inline styles.css]: @toby56 Can you change `overflow` to `overflow-x` and keep it as hidden? Also, please test what happens on long messages - if we put it all in a container `div` can it wrap?

Comment (platy11) [inline styles.css]: @toby56 What can we do to stop the horizontal scrollbars?

**Comment by platy11** [comment]
@tradeandmark/all

**Comment by platy11** [comment]
LGTM :shipit:

**Pull request merged**